veg2hab/habitat.py

```diff
@@ -127,6 +127,22 @@ def valideer_habtype_keuzestatus(cls, values):
 
         return values
 
+    @validator(
+        "opmerking",
+        "mits_opmerking",
+        "mozaiek_opmerking",
+        "debug_info",
+        pre=True,
+        always=True,
+    )
+    def vervang_none_door_lege_string(cls, v):
+        """
+        Omdat ArcGIS niet om kan gaan met lege strings worden deze fields weggeschreven als None
+        Bij het deserializen worden deze dus ook ingelezen als None
+        Dus we zetten ze hier weer om naar een lege string :)
+        """
+        return v if v is not None else ""
```

Comment:
Is dit netter dan het allemaal Optional[str] maken?

Reply:
Ik vind dit wel prima. Al kan het ook prima op dataframe niveau gefixd worden natuurlijk met `fillna` en `replace`.

Reply:
Ging dit nog fout dan?

Reply:
> Ik vind dit wel prima. Al kan het ook prima op dataframe niveau gefixd worden natuurlijk met `fillna` en `replace`.

Op deze manier blijft t ook mooi binnen de class, wat wel charmant is vind ik. Er hoeft van buiten niets mee te gebeuren.

> Ging dit nog fout dan?

Wat fout ging (als ik t me goed herinner) was dat de lege string na het inlezen None werd, wat geen string is, terwijl het volgens de typehints wel een string moet zijn.